### ISSUE TITLE
Can't rezize to a width larger than 701px

### ISSUE THREAD

**Issue by yass-a** [lovell/sharp#2826]
Thank you for the very useful module!
I'm trying to resize an image with Sharp v0.28.3, in an AWS lambda function. The original image has an intrinsic size of 3938 × 5907 px. I call the resize function with the width argument only like this: resize(width).  I pass the width value in the url. The lambda function works with widths below or equal to 701px and it fails with response status 503 when the width is higher. I saw no limit to the width argument's value in the documentation. I wonder what's causing the failure.
Here's the full response : 
_

_> 503 ERROR
> The request could not be satisfied.
> The Lambda function associated with the CloudFront distribution is invalid or doesn't have the required permissions. We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
> If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation._

_

**Comment by lovell** [comment]
This is very, very likely to be a problem with the configuration of your Lambda, and therefore unrelated to sharp.

**Comment by lovell** [comment]
@yass-a Were you able to make any progress with this?

**Comment by yass-a** [comment]
Thank you for asking. I don't know well enough about AWS configs to confirm whether the problem is there. I noticed that the lambda function execution times out after 30 seconds every time I request from Sharp to resize to a value of width x length above 296000. So I decided not to exceed that value and let the browser extend the images when needed. I didn't have enough time for a better solution and I had to move to work on others projects.
